AsyncDisplayKit/ASTableView.mm

```diff
@@ -644,14 +642,14 @@ - (void)tableView:(UITableView *)tableView didEndDisplayingCell:(_ASTableViewCel
   [_rangeController visibleNodeIndexPathsDidChangeWithScrollDirection:self.scrollDirection];
 
   if ([_asyncDelegate respondsToSelector:@selector(tableView:didEndDisplayingNode:forRowAtIndexPath:)]) {
-    ASCellNode *node = ((_ASTableViewCell *)cell).node;
+    ASCellNode *node = cell.node;
     ASDisplayNodeAssertNotNil(node, @"Expected node associated with removed cell not to be nil.");
     [_asyncDelegate tableView:self didEndDisplayingNode:node forRowAtIndexPath:indexPath];
   }
 
   if ([_cellsForVisibilityUpdates containsObject:cell]) {
     [_cellsForVisibilityUpdates removeObject:cell];
-    ASCellNode *node = ((_ASTableViewCell *)cell).node;
```

Comment:
This inert removal of redundant casts was intended for https://github.com/facebook/AsyncDisplayKit/pull/1354 I can revert and move to that PR if desired.